pkg/util/k8s.go

```diff
@@ -148,10 +148,7 @@ func NewClient(configPath string, opts ...Opts) (kubernetes.Interface, error) {
 	)
 	config, err = clientcmd.BuildConfigFromFlags("", configPath)
 	if err != nil {
-		config, err = rest.InClusterConfig()
```

Comment:
BuildConfigFromFlags  function already try to read client config use InCluster mode.

Reply:
This func would only build in-cluster config when `configPath` is empty. So we still need it.

Reply:
ok